### PR TITLE
Add migrations files

### PR DESCRIPTION
**Pull request by Buyaki01**
- Create and run the necessary migration files
- Table and column names should match the ERD diagram
- Foreign keys should be included.
- All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20220326181536_users.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Users < ActiveRecord::Migration[7.0]
+  def change
+    create_table :Users do |t|
+      t.string :Name
+      t.string :Photo
+      t.text :Bio
+      t.timestamps
+      t.integer :PostsCounter
```
Comment (rukundoeric) [inline db/migrate/20220326181536_users.rb]: - [ ] <b>REQUIRED:</b> In rails, it is much better to use `lower case` and `snake case` when naming. For example in this case, `Name` should be `name`,  `Photo` should be `photo`,  `Bio ` should be `bio `,  and `Post_Counter ` should be `post_counter `.

File: db/migrate/20220326181609_posts.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class Posts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :Posts do |t|
+      t.references :User, null: false, foreign_key: true, index: true
+      t.string :Title
+      t.text :Text
+      t.timestamps
+      t.integer :CommentsCounter, default: 0, null: false
+      t.integer :LikesCounter, default: 0, null: false
```
Comment (rukundoeric) [inline db/migrate/20220326181609_posts.rb]: - [ ] <b>REQUIRED:</b> The same thing goes here, Please use `lower case` and `snake case`.

File: db/migrate/20220326181609_posts.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class Posts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :Posts do |t|
+      t.references :User, null: false, foreign_key: true, index: true
```
Comment (rukundoeric) [inline db/migrate/20220326181609_posts.rb]: - [ ] <b>REQUIRED:</b> Remember for User, the project requires to use `author_id`. You can check it out in the given [ERD Diagram.](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) so we do it like this
```rb
t.references :author, foreign_key: { to_table: 'users' }

File: db/migrate/20220326182852_comments.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+class Comments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :Comments do |t|
+      t.references :User, null: false, foreign_key: true, index: true
+      t.references :Post, null: false, foreign_key: true, index: true
+      t.text :Text
```
Comment (rukundoeric) [inline db/migrate/20220326182852_comments.rb]: - [ ] <b>REQUIRED:</b> Remember for User, the project requires to use `author_id`. You can check it out in the given [ERD Diagram.](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) so we do it like this
```rb
t.references :author, foreign_key: { to_table: 'users' } 
```
- [ ] <b>REQUIRED: And also, please user the `lower case` and `snake case`.

File: db/migrate/20220326183004_likes.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+class Likes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :Likes do |t|
+      t.references :User, null: false, foreign_key: true, index: true
+      t.references :Post, null: false, foreign_key: true, index: true
```
Comment (rukundoeric) [inline db/migrate/20220326183004_likes.rb]: - [ ] <b>REQUIRED:</b> Remember for User, the project requires to use `author_id`. You can check it out in the given [ERD Diagram.](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) so we do it like this
```rb
t.references :author, foreign_key: { to_table: 'users' } 
```
- [ ] <b>REQUIRED: And also, please user the `lower case` and `snake case`.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Buyaki01, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by descholar-ceo — APPROVED** [review]
Hi @Buyaki01 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**